### PR TITLE
Speed up screenshoting

### PR DESCRIPTION
**Pull request by mykola-mokhnach**
Actually, the previous screenshoting endpoint was not working properly at all, since it was not returning Base64-encoded screenshot string. Now it's fixed. Also, I've get rid of unnecessary files creation, so the process should be much faster now.

@vmaxim Please take a look when you have time

### CODE REVIEW

File: app/src/main/java/io/appium/uiautomator2/handler/GetElementScreenshot.java [hunk 1]
```diff
@@ -42,27 +43,37 @@ public AppiumResponse safeHandle(IHttpRequest request) {
                 Logger.error("Element is not visible");
                 return new AppiumResponse(getSessionId(request), WDStatus.ELEMENT_NOT_VISIBLE);
             }
-            final Context context = InstrumentationRegistry.getInstrumentation().getTargetContext();
-            final File outputFile = File.createTempFile("screenshot", ".png",
-                    context.getCacheDir());
+            final Bitmap screenshot = uia.takeScreenshot();
```
Comment (vmaxim) [inline app/src/main/java/io/appium/uiautomator2/handler/GetElementScreenshot.java]: Can we extract this dancing with screenshots into separate methods to do `handlers` more readable and avoid code duplication?

Comment (mykola-mokhnach) [reply app/src/main/java/io/appium/uiautomator2/handler/GetElementScreenshot.java]: there is, actually, nothing much to extract. expect of this single line.

File: app/src/main/java/io/appium/uiautomator2/handler/CaptureScreenshot.java [hunk 2]
```diff
@@ -1,17 +1,44 @@
...
+
+    @Nullable
+    private static String takeScreenshot() {
+        final Bitmap screenshot = uia.takeScreenshot();
+        if (screenshot == null) {
+            return null;
+        }
+        try {
+            final ByteArrayOutputStream stream = new ByteArrayOutputStream();
+            if (!screenshot.compress(Bitmap.CompressFormat.PNG, SCREENSHOT_COMPRESSION_QUALITY,
```
Comment (vmaxim) [inline app/src/main/java/io/appium/uiautomator2/handler/CaptureScreenshot.java]: `PNG` is lossless format - quality will be ignored

Comment (mykola-mokhnach) [reply app/src/main/java/io/appium/uiautomator2/handler/CaptureScreenshot.java]: true. Will set to 100

**Pull request merged**